fili-core/src/main/java/com/yahoo/bard/webservice/table/BasePhysicalTable.java

```diff
@@ -3,26 +3,29 @@
 package com.yahoo.bard.webservice.table;
 
 import com.yahoo.bard.webservice.data.config.names.TableName;
-import com.yahoo.bard.webservice.data.dimension.DimensionDictionary;
 import com.yahoo.bard.webservice.data.time.ZonedTimeGrain;
-import com.yahoo.bard.webservice.metadata.SegmentMetadata;
+import com.yahoo.bard.webservice.druid.model.query.Granularity;
 import com.yahoo.bard.webservice.table.availability.Availability;
-import com.yahoo.bard.webservice.table.availability.ImmutableAvailability;
+import com.yahoo.bard.webservice.table.resolver.DataSourceConstraint;
 import com.yahoo.bard.webservice.util.IntervalUtils;
+import com.yahoo.bard.webservice.util.SimplifiedIntervalList;
 
 import org.joda.time.DateTime;
+import org.joda.time.Interval;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.Map;
+import java.util.Set;
 
 import javax.validation.constraints.NotNull;
 
 /**
- * Base Physical Table implements common PhysicalTable capabilities.
+ * Physical Table represents a druid table.
```

Comment:
This comment is also slightly off. 

Now, a `PhysicalTable` _may_ represent a concrete druid (actually we should call it `fact`) table, but it may also represent a view/union/whatevers. It's the `ConcretePhysicalTable` that represents an honest-to-goodness table that you can get your hands on.